diamonds-viewer/.gitignore

```diff
@@ -1,2 +1,23 @@
-node_modules
-build
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
```

Comment:
I think we can change these lines to just *.log. I don't see a situation where we want to commit any log files, do you?